BlueprintUI/Sources/Environment/Keys/WindowSizeKey.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+extension Environment {
+    private enum WindowSizeKey: EnvironmentKey {
+        static var defaultValue: CGSize {
+            UIApplication.shared.windows.first?.bounds.size ?? .zero
+        }
+    }
+
+    /// The size of the window that contains the `BlueprintView`.
+    public var windowSize: CGSize {
```

Comment:
I wonder if this should be optional, and nil by default (and nil if the blueprint view isn't in a window)?

Reply:
I could see an argument for that, but it would make dealing that nil case a pain in elements that use this. Same reasoning probably applies to the other keys that are set by BlueprintView (display scale, layout direction, and safe area insets).